websites/docs/package.json

```diff
@@ -39,7 +39,8 @@
     "eslint": "^8.42.0",
     "fs-extra": "^11.1.1",
     "globby": "^11.1.0",
-    "ts-node": "^10.9.1"
+    "ts-node": "^10.9.1",
+    "utility-types": "^3.10.0"
```

Comment:
`utility-types` should be devDependency of this package. I realized this when I run type:check.